### PR TITLE
Adding openlxp-notifications package to pypi

### PR DESCRIPTION
**Pull request by SHGODE**
@mattdrayer Opening PR for openlxp-notification package

### CODE REVIEW

File: app/setup.cfg [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# setup.cfg
+[metadata]
+name = openlxp-notification
```
Comment (mattdrayer) [inline app/setup.cfg:3]: This should be `openlxp-notifications` right?

**Review by mattdrayer — APPROVED** [review]
Just one question, otherwise looks great!

File: app/setup.cfg (hunk 1 above)
Comment (SHGODE) [reply app/setup.cfg:3]: So earlier I had created the openlxp-notifications 1.0.0 package and later I realized I made small mistakes, so I thought why to make it 1.0.1, I would rather delete openlxp-notifications 1.0.0 and recreated it again. So I deleted the openlxp-notifications 1.0.0 package from PyPI. And when I tried making it again, I got to know the naming policy of PyPI (**https://pypi.org/help/#file-name-reusehttps://pypi.org/help/#file-name-reuse**) it says you can not reuse the same name if you delete that package, you have to create a new package with a different name so now PyPI will not allow openlxp-notifications so then I chose openlxp-notification. I agree it's my mistake I should I read the policies before I deleted the package. 

If you have any other package name suggestions let me know. Sorry about my mistake! :(

Comment (mattdrayer) [reply app/setup.cfg:3]: It looks like we can maybe work with PyPI to correct the situation:  https://github.com/pypa/pypi-support/issues/new?labels=PEP+541&template=pep541-request.yml&title=PEP+541+Request%3A+PROJECT_NAME

**Review by KarenAJ — APPROVED** [review]
Approved

**Pull request merged**